src/components/Profile/components/PersonalInfoList/index.jsx

```diff
@@ -326,11 +330,11 @@ const PersonalInfoList = ({ myProf, profile, isOnline, createSnackbar }) => {
   const advisors =
     (myProf || canViewAcademicInfo) && isStudent ? (
       <ProfileInfoListItem
-        title={profile.Advisors?.length > 1 ? 'Advisors:' : 'Advisor:'}
+        title={advisorsList?.length > 1 ? 'Advisors:' : 'Advisor:'}
         contentText={
-          profile.Advisors?.length < 1
+          advisorsList.length < 1
```

Comment:
No `?` here? I see one above.

Reply:
`advisorList` is initialized as `[]`, the API also returns an empty array if none are found so it's never null